python/ray/serve/handle.py

```diff
@@ -275,6 +275,13 @@ def __reduce__(self):
     def __getattr__(self, name):
         return self.options(method_name=name)
 
+    def shutdown(self):
+        """Shutdown ray serve handle gracefully.
+
+        The router needs to be shutdown separately.
+        """
```

Comment:
Oh, just realized that by default I think this is going to show up in the API reference. To avoid this I think you can prefix it like `_shutdown` (or don't have a docstring)

Reply:
good call, let me drop the docstring real quick